tests/test_s3.py

```diff
@@ -1543,6 +1543,164 @@ def test_has_statement_policy(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
 
+    def test_bucket_replication_policy_remove(self):
+        replicated_from_name = "replication-from-12345"
+        replicated_to_name = "replication-t0-12345"
+
+        self.patch(s3.S3, "executor_factory", MainThreadExecutor)
+        self.patch(
+            s3.SetBucketReplicationConfig, "executor_factory", MainThreadExecutor
+        )
+
+        # Remove GetBucketLocation from augments
+        # was causing AttributeError: 'FakeHttpResponse' object has no attribute 'content'
+        augments = list(s3.S3_AUGMENT_TABLE)
+        augments.remove((
+            "get_bucket_location", "Location", {}, None, 's3:GetBucketLocation'))
+        self.patch(s3, "S3_AUGMENT_TABLE", augments)
+
+        # and ignore any other buckets we might have in this test account
+        # to minimize the placebo data and API calls
+        # Inspired by #5206, thanks tjstansell!
```

Comment:
interesting, and mitigates at least extra copies per augment table, albeit this is client side on the list. source: config is minimal, albeit delayed.